src/containers/JudgingPanel.js

```diff
@@ -202,6 +202,23 @@ export default () => {
     await submitGrade(id, { ...score, removed: true }, { uid: gradeId, email: score.user })
   }
 
+  const onDisqualify = async projectId => {
+    try {
+      await db.runTransaction(async transaction => {
```

Comment:
if this is a single update operation, why do we need a transaction here?

Reply:
Oh I thought we would need a transaction since we are reading and then writing and there could be race conditions. But maybe its better if we just do what the UI shows instead of what the most up to date field is in the backend? (admin isn't subscribed to judging info)

Reply:
i think keeping it consistent with UI makes sense here

Reply:
agreee